onyo/lib/commands.py

```diff
@@ -845,7 +825,7 @@ def onyo_set(inventory: Inventory,
         renaming an asset, while `rename` is not true.
     """
     if not paths:
-        paths = [Path.cwd()]
+        paths = []  # ??
```

Comment:
IMO this if-clause could be removed anyways, and in the function definition, instead of `paths: Optional[list[Path]] = None` just set to an `[]` as default parameter.

Reply:
> IMO this if-clause could be removed anyways, and in the function definition, instead of paths: Optional[list[Path]] = None just set to an [] as default parameter.

Nope. Don't use mutable objects as defaults that way. This list object will change and be re-used during the next call of that function. So, the "default" silently changes between two calls (unless it's something that you only ever read in that function).